### PR TITLE
Headless + indexing

### LINKED ISSUE #5: Current solution for shortening path names doesn't work
If multiple lecture videos have the same name, and are only part numbers are different, the part numbers get cut off and you end up overwriting the file.

### PR DESCRIPTION
**Pull request by dylantjb**
Closes #5

### CODE REVIEW

**Comment by dylantjb** [comment]
Correction: Move .exe from user data dir to the executable path

File: 1_list_videos.py [hunk 1]
```diff
@@ -1,39 +1,46 @@
...
+    print(course)
+    WebDriverWait(driver, 10).until(wait_element)
+    videoDicts = driver.execute_script(open("list_videos.js").read())
+    videos = []
+
+    index = 1
+    for i in range(len(videoDicts) - 1):
+        videoDicts[i]['name'] = str(index) + "_" + videoDicts[i]['name']
+
+        index = index + 1 if videoDicts[i]['week'] == videoDicts[i+1]['week'] else 1
```
Comment (POPLOPUK) [inline 1_list_videos.py:36]: wouldn't this cause an out of bounds error on the last element?

Comment (memst) [reply 1_list_videos.py:36]: Not in this case because the loop's range is off.

File: 1_list_videos.py [hunk 2]
```diff
@@ -1,39 +1,46 @@
...
-from selenium.webdriver.common.by import By
-import time
+import os
 import sqlite3
 
-courses = []
-with open ("courses.txt", "r") as f:
-	courses = [line.strip() for line in f.readlines()]
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as ec
```
Comment (memst) [inline 1_list_videos.py:5]: Selenium's docs style it as EC, so I'd like to leave it that way
https://selenium-python.readthedocs.io/waits.html

**Pull request merged**